python/cudf/cudf/tests/test_datetime.py

```diff
@@ -2107,3 +2107,20 @@ def test_datetime_binop_tz_timestamp(op):
     date_scalar = datetime.datetime.now(datetime.timezone.utc)
     with pytest.raises(NotImplementedError):
         op(s, date_scalar)
+
+
+@pytest.mark.parametrize("data1", [["20110101", "20120101", None]])
+@pytest.mark.parametrize("data2", [["20110101", "20120101", "20130101"]])
```

Comment:
Test all combinations of (left None, right None, both None).
```suggestion
@pytest.mark.parametrize("data1", [["20110101", "20120101", None, "20140101", None]])
@pytest.mark.parametrize("data2", [["20110101", "20120101", "20130101", None, None]])
```